drivers/ieee802154/ieee802154_cc13xx_cc26xx_subg.c

```diff
@@ -434,17 +435,27 @@ static void ieee802154_cc13xx_cc26xx_subg_rx_done(
 {
 	struct net_pkt *pkt;
 	uint8_t len;
-	int8_t rssi;
+	int8_t rssi, status;
 	uint8_t *sdu;
 
 	for (int i = 0; i < CC13XX_CC26XX_NUM_RX_BUF; i++) {
 		if (drv_data->rx_entry[i].status == DATA_ENTRY_FINISHED) {
-			len = drv_data->rx_data[i][1];
-			sdu = &drv_data->rx_data[i][3];
-			rssi = sdu[len - 2];
-			len -= 2;
+			len = drv_data->rx_data[i][0];
+			sdu = drv_data->rx_data[i] + 1;
```

Comment:
technically, without the phy header, that would then become the 'mpdu' rather than the 'psdu'.